frontend/workflows/experimentation/src/list-experiments.jsx

```diff
@@ -9,26 +42,27 @@ const ListExperiments = () => {
       setExperiments(response?.data?.experiments || []);
     });
 
-    return <Table headings={["Downstream Cluster", "Upstream Cluster"]} />;
+    return (
+      <Table headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]} />
+    );
   }
 
   return (
-    <Table
-      data={experiments}
-      headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]}
-    >
-      {experiments.map(e => (
-        <Row
-          key={e.id}
-          data={[
-            e.testSpecification.abort.clusterPair.downstreamCluster,
-            e.testSpecification.abort.clusterPair.upstreamCluster,
-            e.testSpecification.abort.percent,
-            e.testSpecification.abort.httpStatus,
-          ]}
-        />
-      ))}
-    </Table>
+    <Container>
+      <Table
+        data={experiments}
+        headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]}
+      >
+        {experiments.map(e => {
+          if (e.testSpecification.abort) {
+            return renderAbortData(e);
+          }
+          return renderLatencyData(e);
+        })}
+      </Table>
+      <Button onClick={StartAbortExperiment}>Start Abort Experiment</Button>
```

Comment:
@dschaller How do I get these buttons to work? I want them to render the respective wizards

Reply:
You'll have to use react router to navigate to those pages on button click.

Reply:
do you mean like this:

    ReactDOM.render(StartAbortExperiment("test"),
    document.getElementById("root"));

Reply:
we are using react-router so something like:
```jsx
import { useNavigate } from 'react-router-dom';

const navigate = useNavigate();
function handleClick() {
  navigate('/home');
};
```